common4j/src/main/com/microsoft/identity/common/java/nativeauth/providers/NativeAuthRequestProvider.kt

```diff
@@ -224,32 +222,11 @@ class NativeAuthRequestProvider(private val config: NativeAuthOAuth2Configuratio
     //endregion
 
     //region /signup/start
-    internal fun createSignUpStartRequest(
```

Comment:
I don't see the old, now un-used signInStart creation function being removed.

Reply:
This is the method that has been removed.